pkg/controller/compliancescan/scantype.go

```diff
@@ -85,8 +85,10 @@ func (nh *nodeScanTypeHandler) getTargetNodes() ([]corev1.Node, error) {
 	case compv1alpha1.ScanTypePlatform:
 		return nodes.Items, nil // Nodes are only relevant to the node scan type. Return the empty node list otherwise.
 	case compv1alpha1.ScanTypeNode:
+		// we only scan Linux nodes
+		nodeScanSelector := map[string]string{"kubernetes.io/os": "linux"}
```

Comment:
@rhmdnd does EKS contain these node labels?

Reply:
If not, we can pass down the node selector up from the operator level which detects the platform. This is just much simpler.

Reply:
btw upstream kube does have those labels (verified with SPO suite which just installs upstream precompiled binaries)

Reply:
It seems like `"kubernetes.io/os": "linux"` are commonly used

Reply:
https://aws.github.io/aws-eks-best-practices/windows/docs/scheduling/

Reply:
Thanks for following up here @Vincent056